web-marketplace/src/pages/CreditClassDetails/CreditClassDetails.Projects.tsx

```diff
@@ -1,4 +1,6 @@
 import React, { useState } from 'react';
```

Comment:
we use `BuySellOrderFlow` in this file. Do we plan to handle it separately?
good to know that the current flow will be replaced with the fiat/crypto one once that's done, although I'm not sure it'll be fully done before the Terrasos project so maybe still worth it to translate it

Reply:
Yes, it will be handled separately; the first batches of PRs focus only on pages.